app/models/live_metric.rb

```diff
@@ -8,18 +8,20 @@ def self.find(*args)
     validate_raw_query(raw_query)
     processed = process_conditions(raw_query[:conditions])
     resource = fetch_resource(processed[:resource_type], processed[:resource_id])
+    ext_options = raw_query[:ext_options]
+    filtered_cols = ext_options ? ext_options[:only_cols] : nil
```

Comment:
@Fryguy , @lucasponce is out today, can this be merged and he can address the minor issue in another PR?